airbyte-webapp/src/pages/connections/CreateConnectionPage/CreateConnectionPage.tsx

```diff
@@ -251,6 +213,7 @@ export const CreateConnectionPage: React.FC = () => {
               }
             />
           )}
+          {currentStep !== StepsTypes.CREATE_CONNECTION && <InlineEnrollmentCallout withBottomMargin />}
```

Comment:
What is blocking us from having this on the CREATE_CONNECTION step?

Reply:
That's the final step in the flow before hitting the actual connection page, after creating both the source and destination, where you configure name, syncing, etc; it's the one where the page spacing is totally different because for some reason the original author decided to abandon the wrapper element every other step uses for `MainPageWithScroll`, so fixing the alignment means adding some hard-to-understand coupling to the CSS of either `CreateConnectionPage` or `InlineEnrollmentCallout`.

Reply:
Strictly speaking, this check is redundant, because of lines 189-195 above:
```ts
  if (currentStep === StepsTypes.CREATE_CONNECTION) {
    return (
      <MainPageWithScroll headTitle={headTitle} pageTitle={pageHeader}>
        {renderStep()}
      </MainPageWithScroll>
    );
  }
```
but I think it's best to leave the redundant check in to make regressions less likely in the inevitable refactoring: the question is not whether this component will be rewritten, but how soon and by whom.

Reply:
Hah, in fact, the redundant update is now being reported as a typescript error, so I'll remove it anyway.